packages/notebook/src/widget.ts

```diff
@@ -1284,10 +1292,42 @@ export class Notebook extends StaticNotebook {
    * Construct a notebook widget.
    */
   constructor(options: Notebook.IOptions) {
-    super(options);
+    super({
+      renderer: {
+        createOuter(): HTMLElement {
+          return document.createElement('div');
+        },
+
+        createViewport(): HTMLElement {
+          const el = document.createElement('div');
+          el.setAttribute('role', 'feed');
+          el.setAttribute('aria-label', 'Cells');
+          return el;
+        },
+
+        createScrollbar(): HTMLOListElement {
+          return document.createElement('ol');
+        },
+
+        createScrollbarItem(
+          index: number,
+          model: ICellModel,
+          notebook: Notebook
+        ): HTMLLIElement {
+          const li = document.createElement('li');
+          li.appendChild(document.createTextNode(`${index + 1}`));
+          if (notebook.selectedCells.some(cell => model === cell.model)) {
+            li.className = 'jp-mod-selected';
+          }
```

Comment:
Should we distinguish between selected and active?

Reply:
I think that is a good idea and we should do it. We should probably add a lot of semantic information to this scrollbar over time and have it approach being a document mini-map. Do you think it is necessary for a minimum viable feature?

Reply:
Added the CSS modifier and an initial style for `selected` and also for `active`.